src/routes/(app)/documents/DeleteFileForm.svelte

```diff
@@ -21,16 +22,17 @@
 </button>
 <dialog bind:this={deleteModal} class="modal modal-bottom sm:modal-middle">
   <div class="modal-box">
-    <h3 class="text-lg font-bold">Delete file</h3>
+    <h3 class="text-lg font-bold">{m.documents_deleteFile()}</h3>
     <p class="py-4">
-      Are you sure you want to delete the file <strong>{fileName}</strong>?
+      <!-- eslint-disable-next-line svelte/no-at-html-tags -->
+      {@html m.documents_deleteAreYouSure({ fileName })}
```

Comment:
I don't like excessive use of @html if it can be avoided. Is there no paraglide component to handle inline styling of texts? Some other i18n libs I've used before (like i18n-react) has support for custom inline styling. This isn't a big problem, as the text is not user submitted.

I think a merge can still be done but just thought I'd leave this comment here.